frontendservice/src/router/index.js

```diff
@@ -80,6 +81,14 @@ const router = new VueRouter({
       component: Verify,
       meta: {
         requiresUnverified: true,
+      }
+    },
+    {
+      path: "/shop-owner/createshop",
+      name: "CreateShop",
+      component: CreateShop,
+      meta: {
+        requiresLogin: true,
```

Comment:
According to this logic, every shop-owner (who is logged in) will be able to access this particular link and create a new shop (as far as I can understand, correct me if I'm wrong). What we want is that the shop-owner should be verified in order to access this feature. Is this particular case being checked by your code?

Reply:
Add this - 

else if (requiresVerified && !verified) {
     if (currentUser) next("shop-owner");
     else next("customer");
}

Reply:
Added, Thanks!